engine/gamesys/src/gamesys/scripts/script_buffer.cpp

```diff
@@ -106,6 +106,7 @@ namespace dmGameSystem
 #define SCRIPT_LIB_NAME "buffer"
 #define SCRIPT_TYPE_NAME_BUFFER "buffer"
 #define SCRIPT_TYPE_NAME_BUFFERSTREAM "bufferstream"
+#define SCRIPT_TYPE_NAME_BUFFERMETADATA "buffermetadata"
```

Comment:
I didn't notice this last time.
There is no Lua UserData type called "buffermetadata", so I think the use of this in the same fashion as the other (actual) types is confusing. I think it's better to just write it explicitly for those messages: "metadata %s doesn't exist in the buffer"

Reply:
ok, i edited all error messages that used "buffermetadata".